sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -2362,6 +2365,106 @@ test_suite(
     tests = ["epmp_init_otp_" + lc_state for lc_state, _ in get_lc_items()],
 )
 
+platform_target(
+    name = "rom_with_fake_keys_fpga_cw310_elf",
+    testonly = True,
+    platform = OPENTITAN_PLATFORM,
+    target = "//sw/device/silicon_creator/rom:rom_with_fake_keys_fpga_cw310.elf",
+)
+
+rust_binary(
+    name = "openocd_debug_test",
+    srcs = [
+        "openocd_debug_test.rs",
+    ],
+    deps = [
+        "//hw/top_earlgrey/sw/autogen/chip:top_earlgrey",
+        "//sw/host/opentitanlib",
+        "//third_party/rust/crates:regex",
+        "@crate_index//:anyhow",
+        "@crate_index//:log",
+        "@crate_index//:object",
+        "@crate_index//:structopt",
+    ],
+)
+
+[
```

Comment:
You can merge those two test lists and conditionally add `--expect-fail` like so:
```python
LC_STATES_DEBUG_DISALLOWED = [CONST.LCV.PROD, CONST.LCV.PROD_END]
...
test_cmds = [...] +(["--expect-fail"] if lc_state in LC_STATES_DEBUG_DISALLOWED else [])
```